WMass/python/plotter/fakeRate.cc

```diff
@@ -79,6 +79,8 @@ float fakeRateWeight_1l_i_smoothed(float lpt, float leta, int lpdgId, bool passW
     int etabin = std::max(1, std::min(hist->GetNbinsX(), hist->GetXaxis()->FindBin(feta)));
     float p0 = hist->GetBinContent(etabin, 1);
     float p1 = hist->GetBinContent(etabin, 2);
+    if (iFR==1) p0 += hist->GetBinError(etabin, 1);
+    if (iFR==2) p0 -= hist->GetBinError(etabin, 1);
```

Comment:
apart the electron-only changes, this is a change in the common part. 
Not sure it is the best choice, but it is a first try to propagate a systematic on FR: since the FR is a line, cover a region by shifting +/- 1 sigma the offset (not changing the slope)